workload/rockcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: catalogue 
+summary: A service catalogue for workloads running in Juju.
+description: A service catalogue for workloads running in Juju.
+version: "0.15.0"
+base: ubuntu:22.04
```

Comment:
When would we use `bare`?

Reply:
The same as `scratch` in Docker|Containerfiles. If you have a single statically compiled binary which doesn't link against anything in libc. Prometheus, for example, easily could be if we wanted it to be.